causalpy/pymc_models.py

```diff
@@ -270,12 +285,7 @@ def build_model(self, X, y, coords):
             n_predictors = X.shape[1]
             X = pm.Data("X", X, dims=["obs_ind", "coeffs"])
             y = pm.Data("y", y[:, 0], dims="obs_ind")
-            # TODO: There we should allow user-specified priors here
             beta = pm.Dirichlet("beta", a=np.ones(n_predictors), dims="coeffs")
```

Comment:
So this is the component that would be updated with multiple treatment units? Allowing a matrix of n_predictors x n_targets?

Reply:
Yes. Plan was to do things stepwise, so this PR is about API which allows for the next PR which allows multiple treated units. Just to keep things bite-sized etc.